src/test/java/com/fittracker/fittracker/controller/AuthenticationControllerIntegrationTest.java

```diff
@@ -44,112 +39,82 @@ protected String getEndpoint() {
         return ENDPOINT;
     }
 
+    private <T> T makeUnauthorizedPostRequest(String endpoint, Object requestBody, HttpStatus expectedStatus,
+        Class<T> responseClass) throws Exception {
+        var responseString = mockMvc.perform(post(create(endpoint))
+                .contentType(APPLICATION_JSON)
+                .content(mapper.writeValueAsString(requestBody)))
+            .andExpect(status().is(expectedStatus.value()))
+            .andReturn().getResponse().getContentAsString();
+
+        return mapper.readValue(responseString, responseClass);
+    }
```

Comment:
What about putting this at the end of a class as it is a helper method?

Reply:
yes, should be at the end, looks like formatter moved it to the top. fixed